Week3/homework/step2-2.js

```diff
@@ -2,19 +2,27 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  numbers.forEach(number => {
+    if (number % 15 === 0) {
+      threeCallback(number);
+      fiveCallback(number);
+    } else if (number % 3 === 0) {
+      threeCallback(number);
+    } else if (number % 5 === 0) {
+      fiveCallback(number);
+    }
+  });
 }
 
 function sayThree(number) {
-  // Replace this comment and the next line with your code
-  console.log(number);
+  console.log(number + ': divisible by 3');
```

Comment:
You might be interested to know that inside console.log you can separate parameters by commas. So you could write it like this, 
`console.log(number, ': divisible by 3');`
I often find using commas makes it easier - so you can log multiple things on a single line.